base_import_match/models/base_import.py

```diff
@@ -88,7 +89,7 @@ def _compute_name(self):
                 " + ".join(
                     s.field_ids.mapped(
                         lambda r: (
-                            str(r.field_id.name) +
+                            unicode(r.field_id.name) +
```

Comment:
I mean,

> odoo returns a unicode string then we don't need a `unicode(string_unicode)` again...
 for all strings returned.

We could use `string_unicode or 'None'` just for None cases avoiding additional re-processing of `unicode` method.